browser/app/js/buckets/BucketList.js

```diff
@@ -39,6 +40,7 @@ export class BucketList extends React.Component {
             ))}
           </ul>
         </Scrollbars>
+        <MakeBucketModal />
```

Comment:
Is there any specific reason why MakeBucketModal is inside BucketList, whereas UploadModal is inside MainContent? Do you think it would be better if MakeBucketModal was also inside MainContent?

Reply:
There is no specific reason, just wanted to keep it closer to buckets. We can move it to MainContent alongside UploadModal.

Reply:
Ok, I think we can keep it here for practicality then